src/utils/internal/jsonParse.test.ts

```diff
@@ -11,3 +11,7 @@ test('returns undefined without an error given an invalid JSON string', () => {
   expect(parse).not.toThrow()
   expect(parse()).toBeUndefined()
 })
+
+test('parse a single number as valid JSON string', () => {
+  expect(jsonParse('1')).toEqual(1)
```

Comment:
Is there a practical benefit of adding this test? `jsonParse` is a try/catch wrapper around native `JSON.parse`. What we're doing here is effectively testing how the latter behaves, which shouldn't be our responsibility (asserting that it indeed parses a valid stringified object in the first test case is sufficient). I think we should remove this test, as it's not connected with any change.